Agreements/External/EE-N_15.txt

```diff
@@ -0,0 +1 @@
+FIR_COPX:*:*:MOTOX:*:*:LFEE N:London S15:*:*:MOTOX
```

Comment:
I'm not sure why we had this really... though we could perhaps do with an FIR_COPX point for IRKUN into TC SE?

Reply:
as well as? or in place of?

Reply:
In place of, in a separate file. Northbound traffic via MOTOX isn't permitted from France to the UK. But it might be worth adding FIR COPX for the various possible low level routes into TC SE?
![image](https://user-images.githubusercontent.com/12017284/115232724-fa8b9d80-a10e-11eb-804a-a108f2e0a68c.png)

Reply:
done